pom.xml

```diff
@@ -138,6 +138,7 @@
 		<dependency>
 			<groupId>baseCode</groupId>
 			<artifactId>baseCode</artifactId>
+			<version>1.1.5</version>
```

Comment:
This can be reverted once merged in pavlab-starter-parent. https://github.com/PavlidisLab/pavlab-starter-parent/issues/14